src/modules/command-registry.ts

```diff
@@ -52,6 +52,7 @@ let _cachedConfig: CommandRegistryConfiguration = {};
 // -----------------------------------------------------------------------------------------
 
 const CommandRegistry = {
+    ALIAS_PREFIX,
```

Comment:
Minor: since there are comments directly below this, I could see it having a line feed after it